osgp/platform/osgp-adapter-domain-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/domain/da/application/config/messaging/InboundOsgpCoreRequestsMessagingConfig.java

```diff
@@ -50,4 +52,9 @@ public DefaultMessageListenerContainer messageListenerContainer(
         return this.jmsConfigurationFactory.initMessageListenerContainer(messageListener);
     }
 
+    @Bean(name = "domainDistributionAutomationInboundOsgpCoreRequestsaMessageProcessorMap")
+    public MessageProcessorMap messageProcessorMap() {
+        return new BaseMessageProcessorMap("domainDistributionAutomationInboundOsgpCoreRequestsaMessageProcessorMap");
```

Comment:
Typo in bean name and constructor argument: domainDistributionAutomationInboundOsgpCoreRequests**a**MessageProcessorMap

Reply:
Fixed